tests/client-dbus/README.rst

```diff
@@ -30,9 +30,6 @@ dbus-signature-pyparsing/src
 To run only the udev tests: ::
    > make udev-tests
 
-To run only the miscellaneous tests: ::
-   > make misc-tests
-
```

Comment:
I think we should delete the README entirely. Right here I'm just deleting a thing which is ultra crufty from long disuse and which is related to the test I'm adding back in. I think that since we are now, with this PR, back to running all the tests either in TMT or GitHub workflows, that how to run them is pretty well documented, also their dependencies, etc. I'll go ahead and add a commit to remove the README entirely.